sdk-blueprints/update-schema.sh

```diff
@@ -0,0 +1,22 @@
+# Internal script for debugging SDK generation locally.
+# Run from the root of the openapi-clients repository.
+
+# Define directory where projects are stored
+projects_dir="$HOME/projects"
+
+vserver_dir="$projects_dir/com.visier.vserver"
+python_sdk_repository_dir="$projects_dir/python-sdk/src"
+
+version="22222222.99201.1494"
```

Comment:
Should not have hard-coded versions in the file. Use `$1` and error if not provided

Reply:
I added version as a parameter, but set 0.0.001 version as default to make this script easy to call.